examples/can-arcade-drive/robot.py

```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2018 FIRST. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+# ----------------------------------------------------------------------------
+
+import rev
+import wpilib
+from wpilib.drive import DifferentialDrive
+
+
+class Robot(wpilib.TimedRobot):
+    # SPARK MAX controllers are intialized over CAN by constructing a
+    # CANSparkMax object
+    #
+    # The CAN ID, which can be configured using the SPARK MAX Client, is passed
+    # as the first parameter
+    #
+    # The motor type is passed as the second parameter.
+    # Motor type can either be:
+    #   rev.MotorType.kBrushless
+    #   rev.MotorType.kBrushed
+    #
+    # The example below initializes four brushless motors with CAN IDs
+    # 1, 2, 3 and 4. Change these parameters to match your setup
+
+    def robotInit(self):
+        # Create motors
+        self.leftLeadMotor = rev.CANSparkMax(1, rev.MotorType.kBrushless)
+        self.rightLeadMotor = rev.CANSparkMax(0, rev.MotorType.kBrushless)
+        self.leftFollowMotor = rev.CANSparkMax(2, rev.MotorType.kBrushless)
+        self.rightFollowMotor = rev.CANSparkMax(3, rev.MotorType.kBrushless)
```

Comment:
The comment above lists CAN IDs 1, 2, 3, 4. Is there a reason why these are different?

Also, the comment above should be next to this code like it is in REV's examples.

Reply:
Sorry about that. Usually, when I configure my robots, their CAN IDs are zero-based.